src/main/java/jenkins/plugins/publish_over_ssh/BapSshHostConfiguration.java

```diff
@@ -42,51 +46,71 @@
 import org.apache.commons.logging.LogFactory;
 import org.kohsuke.stapler.DataBoundConstructor;
 
-import com.jcraft.jsch.ChannelSftp;
-import com.jcraft.jsch.JSch;
-import com.jcraft.jsch.JSchException;
-import com.jcraft.jsch.Session;
-import com.jcraft.jsch.SftpException;
-
 @SuppressWarnings("PMD.TooManyMethods")
 public class BapSshHostConfiguration extends BPHostConfiguration<BapSshClient, BapSshCommonConfiguration>
-                                                                                        implements Describable<BapSshHostConfiguration> {
+        implements Describable<BapSshHostConfiguration> {
 
     private static final long serialVersionUID = 1L;
     public static final int DEFAULT_PORT = 22;
     public static final int DEFAULT_TIMEOUT = 300000;
     public static final String CONFIG_KEY_PREFERRED_AUTHENTICATIONS = "PreferredAuthentications";
     private static final Log LOG = LogFactory.getLog(BapSshHostConfiguration.class);
+    public static final String HTTP_PROXY_TYPE = "http";
```

Comment:
does not seem to be definied neither in `com.jcraft.jsch.Proxy` or sibling classes, nor in `java.net.Proxy`